tests/endpoints/Files.test.js

```
@@ -98,6 +98,32 @@ describe("files", () => {
       ]);
     });
 
+    test("api - commit", async () => {
+      mockAPI(
+        "/projects/project-id/branches/branch-id/commits/commit-id/files",
```

Heyo @amccloud! Could you please describe how current testing works? E.g. what project-id, or branch-id it uses?

---

@Nivl hi! maybe you can help me? What project-id and branch-id gets used in this case?

---

@berezovskycom Keep in mind this is no a end-to-end test of backend services and such. The responsibility of the sdk is to make sure it calls the api correctly and formats the returned data correctly. We accomplish this test by mocking the expected request from the sdk with the expected response

As a convention we use the name of the property is used as a placeholder value for testing. `branch-id` and `project-id` are valid ids and are a bit easier to debug and understand during test failures compared to random UUIDs